cloudinstall/single_install.py

```diff
@@ -178,7 +223,8 @@ def _install_upstream_deb(self):
         log.debug('Found upstream deb, installing that instead')
         filename = os.path.basename(self.config.getopt('upstream_deb'))
         utils.container_run(
-            self.container_name, 'sudo dpkg -i .cloud-install/{}'.format(
+            self.container_name,
+            'sudo dpkg -i /home/ubuntu/.cloud-install/{}'.format(
```

Comment:
Nice catch. Thanks! I guess I mildly prefer not using ssh here since it
should be slightly faster, but it's not like I measured that. It might not
be noticeable.

On Thursday, January 22, 2015, adam.stokes <notifications@github.com> wrote:

> In cloudinstall/single_install.py
> <https://github.com/Ubuntu-Solutions-Engineering/openstack-installer/pull/366#discussion_r23427404>
> :
>
> > @@ -178,7 +223,8 @@ def _install_upstream_deb(self):
> >          log.debug('Found upstream deb, installing that instead')
> >          filename = os.path.basename(self.config.getopt('upstream_deb'))
> >          utils.container_run(
> > -            self.container_name, 'sudo dpkg -i .cloud-install/{}'.format(
> > +            self.container_name,
> > +            'sudo dpkg -i /home/ubuntu/.cloud-install/{}'.format(
>
> Can either do this way or use_ssh=True and keep it relative. I dont really
> have a preference which.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Ubuntu-Solutions-Engineering/openstack-installer/pull/366/files#r23427404>
> .
>